jsonnetsecure/jsonnet_processvm.go

```diff
@@ -0,0 +1,83 @@
+// Copyright © 2022 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package jsonnetsecure
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+func NewProcessVM(opts *vmOptions) VM {
+	return &ProcessVM{
+		path: opts.jsonnetBinaryPath,
+		args: opts.args,
+		ctx:  opts.ctx,
+	}
+}
+
+func (p *ProcessVM) EvaluateAnonymousSnippet(filename string, snippet string) (string, error) {
+	ctx, cancel := context.WithTimeout(p.ctx, 1*time.Second)
+	defer cancel()
+
+	var (
+		stdin          bytes.Buffer
+		stdout, stderr strings.Builder
+	)
+	p.params.Filename = filename
+	p.params.Snippet = snippet
+
+	if err := p.params.EncodeTo(&stdin); err != nil {
+		return "", errors.WithStack(err)
+	}
+
+	cmd := exec.CommandContext(ctx, p.path, p.args...) //nolint:gosec
+	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
```

Comment:
This is not compatible with Windows, see CI failure @hperl :

https://github.com/ory/x/actions/runs/3416683308/jobs/5687058093#step:5:218

Reply:
Ah right, turns out we don't need it at all.